IDE/Espressif/include.am

```diff
@@ -0,0 +1,23 @@
+# vim:ft=automake
+# included from Top Level Makefile.am
+# All paths should be given relative to the root
+
+EXTRA_DIST+= IDE\Espressif\CMakeLists.txt
+EXTRA_DIST+= IDE\Espressif\partitions_singleapp_large.csv
+EXTRA_DIST+= IDE\Espressif\README.md
+
+# wolfSSL source code is not included here and must be available in separate directory.
+EXTRA_DIST+= IDE\Espressif\components\wolfssl\CMakeLists.txt
+EXTRA_DIST+= IDE\Espressif\components\wolfssl\include\config.h
+EXTRA_DIST+= IDE\Espressif\components\wolfssl\include\user_settings.h
+
+$ the wolfTPM source code used will typically be the parent on the IDE directory.
+EXTRA_DIST+= IDE\Espressif\components\wolftpm\CMakeLists.txt
+EXTRA_DIST+= IDE\Espressif\components\wolftpm\include\options.h
+
+# The example application.
```

Comment:
Need to add IDE/Espressif/main/Kconfig.projbuild here?

Reply:
yes, thanks for that one.